pupil-spa/src/app/virtual-keyboard/virtual-keyboard.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="question-container">
+  <form id='js-question-form' action="/check/question/{{questionNum}}" method="POST" autocomplete="off">
+    <input type="hidden" id="js-form-answer" name="answer" value="">
```

Comment:
There is no need for these hidden inputs is there?  All state will kept in the Components and models.